content/docs/rfcs/14/README.md

```diff
@@ -82,11 +94,17 @@ message WakuMessage {
   optional bool ephemeral = 31;
 }
 ```
+## Waku Message Vaildator
 
-An example proto file following this specification can be found [here (vacp2p/waku)](https://github.com/vacp2p/waku/blob/main/waku/message/v1/message.proto).
```

Comment:
This link is not necessary as the message.proto  is displayed in the specification.